### DISCUSSION TITLE
I made a file conversion tool using Crank.JS

### DISCUSSION THREAD

**Discussion by kristiandupont** [bikeshaving/crank#275] [Show and tell]
I've played a little bit with Crank before and find the concept very intriguing.
I decided to try it out on this silly project that I wanted to make for a while: an in-browser file conversion tool. Basically, I wanted to be able to convert .wav files to .mp3. I have FFmpeg installed but I keep forgetting the CLI options and the online services send my files to a server which I didn't want. So I made this: https://github.com/kristiandupont/concert that runs webasm versions of FFmpeg and ImageMagick to convert files directly in the browser. It's very 0.1 but Crank was great for it!

**Comment by brainkim** [comment]
This is really cool and immediately useful. I love the background Wave component. Thanks for reaching out.

**Reply by kristiandupont** [reply]
Thank you :-) 
I still want to create a larger project that needs proper state management to see how I feel about this paradigm but there is definitely something there. Thank you for creating the library!
